app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
I believe it's deprecated https://github.com/heartcombo/devise/issues/5257

You probably should use partial to override styles here